lib/IRGen/GenMeta.cpp

```diff
@@ -1661,6 +1662,20 @@ namespace {
       for (auto *prop : properties)
         IGM.IRGen.noteUseOfTypeMetadata(prop->getValueInterfaceType());
     }
+
+    void addObjCResilientClassStubInfo() {
+      if (IGM.getClassMetadataStrategy(getType()) !=
+            ClassMetadataStrategy::Resilient)
+        return;
+
+      if (!hasObjCResilientClassStub(IGM, getType()))
+        return;
```

Comment:
Why is the first condition not part of what the second function checks?  I could see if this is a definition-vs-use situation, but we're always at the definition site here, right?

Reply:
For some reason I factored it so that `hasObjCResilientClassStub()` *asserts* the first condition here. I guess it makes sense to change it around.